javascript.js

```diff
@@ -0,0 +1,77 @@
+var dropdown = document.getElementById("dropdown");
+var dropdownlist = document.getElementById("dropdown__list");

```

Comment:
Conventia la numele de variabile ar fi camel case. ex: dropdownList
mai multe aici:
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841